l3backend/build.lua

```diff
@@ -9,7 +9,7 @@ bundle = ""
 -- Location of main directory: use Unix-style path separators
 maindir = ".."
 
-installfiles = {"*.def", "*.pro"}
+installfiles = {"*.def", "*.pro", "*.lua"}
```

Comment:
You need to skip `build.lua` ;) (Perhaps we should look at some `l3build` feature to avoid that, `suppressfiles` or similar?)

Reply:
Or support negative patterns with prefix `!`, like how it's used in [`.gitignore` files](https://git-scm.com/docs/gitignore#_pattern_format).

Reply:
@zauguin  Thoughts on the suggestion? Might be the best plan longer term

Reply:
I agree that this would make sense, but we might want to additionally treat `build.lua` specially since that's a file which will come up in rather many l3build based projects.

Reply:
indeed, that one should be ignored by default